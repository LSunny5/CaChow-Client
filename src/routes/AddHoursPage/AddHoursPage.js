import React, { Component } from 'react';
import HoursForm from '../../components/Hours/HoursForm';
import '../../components/Hours/HoursForm.css';

export default class AddHoursPage extends Component {
    render() {
        return (
            <section className='HFormContent'>
                <h2 className="formTitle">Add Restaurant Hours</h2>
                <HoursForm nextLocation="/restaurants/add" />
            </section >
        )
    };
}