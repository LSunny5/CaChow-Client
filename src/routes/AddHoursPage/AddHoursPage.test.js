import React from 'react';
import ReactDOM from 'react-dom';
import AddHoursPage from './AddHoursPage';
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import { BrowserRouter } from 'react-router-dom';

describe(`AddHoursPage route`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <AddHoursPage />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the AddHoursPage form', () => {
        const wrapper = shallow(<AddHoursPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
});