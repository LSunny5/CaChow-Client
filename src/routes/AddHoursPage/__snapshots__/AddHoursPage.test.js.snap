// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddHoursPage route renders the AddHoursPage form 1`] = `
<section
  className="HFormContent"
>
  <h2
    className="formTitle"
  >
    Add Restaurant Hours
  </h2>
  <HoursForm
    nextLocation="/restaurants/add"
  />
</section>
`;
